src/ServiceProvider.php

```diff
@@ -45,27 +48,26 @@ public function boot()
     {
         parent::boot();
 
-        $this->loadViewsFrom(__DIR__.'/../resources/views', 'runway');
-        $this->mergeConfigFrom(__DIR__.'/../config/runway.php', 'runway');
+        $this->loadViewsFrom(__DIR__ . '/../resources/views', 'runway');
+        $this->mergeConfigFrom(__DIR__ . '/../config/runway.php', 'runway');
 
-        if (! config('runway.disable_migrations')) {
-            $this->loadMigrationsFrom(__DIR__.'/../database/migrations');
+        if (!config('runway.disable_migrations')) {
+            $this->loadMigrationsFrom(__DIR__ . '/../database/migrations');
         }
 
         $this->publishes([
-            __DIR__.'/../config/runway.php' => config_path('runway.php'),
+            __DIR__ . '/../config/runway.php' => config_path('runway.php'),
         ], 'runway-config');
 
         Statamic::booted(function () {
             Runway::discoverResources();
 
             $this->registerPermissions();
+            $this->registerSearchables();
             $this->bootGraphQl();
 
-            if (Runway::usesRouting()) {
-                $this->app->get(\Statamic\Contracts\Data\DataRepository::class)
-                    ->setRepository('runway-resources', Routing\ResourceRoutingRepository::class);
-            }
+            $this->app->get(\Statamic\Contracts\Data\DataRepository::class)
```

Comment:
Just for note: I've made the `ResourceRoutingRepository` load for every site, even if no resources have routing enabled.

Before/after we merge this, it'll probably end up being renamed to the `ResourceRepository` instead.